datacite-draft-doi-handler/src/test/resources/doi_request_event.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Filename, is this input I believe for your handler, please make a hint about it in the filename